jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql5-cluster-jbpm-schema.sql

```diff
@@ -0,0 +1,854 @@
+    create table Attachment (
```

Comment:
to follow the same naming convention and to avoid any potential confusion I would rename the file to `mysql-innodb-cluster-jbpm-schema.sql`

Reply:
chnaged